services/bundlephobia/bundlephobia.service.js

```diff
@@ -27,78 +42,31 @@ module.exports = class Bundlephobia extends LegacyService {
       {
         title: 'npm bundle size',
         pattern: ':format(min|minzip)/:packageName',
```

Comment:
Looking at this on the staging app, we're missing some of the params in the front-end:

![screenshot at 2019-03-02 21-58-31](https://user-images.githubusercontent.com/6025893/53688106-9fcc3780-3d36-11e9-9053-d74d9ace5df3.png)

It might just be a matter of removing this line, or maybe we need to provide several examples in order to cover the scope/version cases

Reply:
That's the experience we currently have in prod today, but agreed on adding/updating examples 👍

Reply:
I tried the single example with no pattern overriding, but the scope param kept crashing things. Added a few different examples accordingly:

![image](https://user-images.githubusercontent.com/13042488/53698842-94652480-3da7-11e9-98d9-ae1c5e143afc.png)